package.json

```diff
@@ -93,7 +93,8 @@
     "semantic-release": "^17.1.1",
     "shelljs": "0.8.4",
     "showdown": "1.9.1",
-    "shx": "0.3.2"
+    "shx": "0.3.2",
+    "typescript": "^4.0.3"
```

Comment:
Stencil <2 had a dependency on typescript. That dependency has been removed, which meant that other dependencies we have failed because typescript wasn't available. So I had to add this dev-dependency.

Reply:
Some thoughts... (not really related to this PR :smile: )
Should we not be able to put almost _all_ of our dependencies as dev-dependencies? For a consumer that installes lime-elements, all the dependencies will be installed as well. That should not be needed since we compile all our code into a distributable package :thinking: Everything listed here in `package.json` is only for building lime-elements, not consuming it :slightly_smiling_face:

Reply:
Almost all our dependencies are dev-dependencies… 😄

Reply:
Yeah but I think most if not all of the other ones probably could be as well :slightly_smiling_face:

Reply:
```json
  "dependencies": {
    "@limetech/material-components-web": "^4.0.1-p4.0.0.1",
    "@popperjs/core": "^2.5.3",
    "@types/tabulator-tables": "^4.7.1",
    "ajv": "^6.12.2",
    "awesome-debounce-promise": "^2.1.0",
    "chart.js": "^2.9.3",
    "flatpickr": "^4.6.3",
    "html-escaper": "^3.0.0",
    "jsx-dom": "^6.4.17",
    "kompendium": "^0.7.0",
    "moment": "^2.26.0",
    "number-abbreviate": "^2.0.0",
    "react-jsonschema-form": "^1.8.1",
    "react-shadow-dom-retarget-events": "^1.0.11",
    "tabulator-tables": "^4.8.2"
  }
```

Really? `kompendium` should be a dev-dependency, but apart from that, it would seem to me like all of these are needed by the final build? Unless of course the tree-shaking pulls in and embeds any code that's used… 🤔

Hmm… yeah, I guess it would be worth a try to just move everything to dev-deps and see if it works 👍

Different branch though, obviously.

Reply:
Yeah I think all of those will be compiled into our package. No need for consumers of lime-elements to install those :smile: Just a theory though :smile: